public/pages/Snapshots/helper.tsx

```diff
@@ -28,3 +29,39 @@ export function snapshotStatusRender(value: string): React.ReactElement {
 
   return <EuiHealth color={color}>{capital}</EuiHealth>;
 }
+
+
+export const getToasts = (id: string, message: string | undefined, snapshotId: string, onClick: (e: React.MouseEvent) => void): Toast[] => {
+  const toasts = [
+    {
+      id: "success_restore_toast",
+      title: `Restore from snapshot "${snapshotId}" is in progress.`,
+      iconType: "check",
+      color: "success",
+      text: (
+        <>
+          <EuiSpacer size="xl" />
+          <EuiFlexGroup justifyContent="flexEnd" style={{ paddingRight: "1rem" }}>
+            <EuiButton onClick={onClick}>View restore activities</EuiButton>
+          </EuiFlexGroup>
+        </>
+      )
+    },
+    {
+      id: "error_restore_toast",
+      title: `Failed to restore snapshot "${snapshotId}"`,
+      color: "danger",
+      text: (
+        <>
+          <EuiText size="s">{message}</EuiText>
+          <EuiSpacer size="xl" />
+          <EuiFlexGroup justifyContent="flexEnd" style={{ paddingRight: "1rem" }}>
+            <EuiButton onClick={onClick} color="danger">View full error</EuiButton>
+          </EuiFlexGroup>
+        </>
+      )
+    }
+  ]
+
+  return id === "success_restore_toast" ? [toasts[0]] : [toasts[1]];
```

Comment:
nit: maybe just a if else is more clear here

Reply:
Fair enough.

Reply:
done